data/formatters/generic.yaml

```diff
@@ -150,6 +168,113 @@ short_source: 'HIST'
 source: 'Zsh Extended History'
 ---
 type: 'conditional'
+data_type: 'skydrive:log:old:line'
+message:
+- '[{source_code}]'
+- '({log_level})'
+- '{text}'
+short_message:
+- '{text}'
+short_source: 'LOG'
+source: 'SkyDrive Log File'
+---
+type: 'conditional'
+data_type: 'skydrive:log:line'
+message:
+- '[{module}'
+- '{source_code}'
+- '{log_level}]'
+- '{detail}'
+short_message:
+- '{detail}'
```

Comment:
Just noticed the [] got dropped from the previous formatter, is this WAI?

Reply:
Same in other short_messages

Reply:
Not sure:
* module and log_level contain `[]`
* the other `[]` are for the lists

Could you double check/confirm

```
class SkyDriveLogFormatter(interface.ConditionalEventFormatter):	
  """Formatter for a SkyDrive log file event."""	

  DATA_TYPE = 'skydrive:log:line'	

  FORMAT_STRING_PIECES = [	
      '[{module}',	
      '{source_code}',	
      '{log_level}]',	
      '{detail}']	

  FORMAT_STRING_SHORT_PIECES = ['{detail}']	

  SOURCE_LONG = 'SkyDrive Log File'	
  SOURCE_SHORT = 'LOG'
```

Reply:
oh right I see now, was confused

Reply:
noworries, thx for confirming